src/app/components/reports/pages/event-summary/new/new.component.html

```diff
@@ -0,0 +1,105 @@
+<h1 class="text-muted">{{ componentTitle }}</h1>
+<h1 class="mb-5">New Event Summary Report</h1>
+
+<form #form="ngForm">
+  <baw-date-time-filter
+    (modelChange)="updateViewModelFromDateTimeModel($event)"
+  ></baw-date-time-filter>
+
+  <!-- We disable the input and use a placeholder value if the form is immutably scoped to a region/site -->
+  <baw-typeahead-input
+    *ngIf="!site || site?.isPoint"
+    label="Site(s)"
+    [inputPlaceholder]="region ? region.name : 'All'"
+    [searchCallback]="regionsSearchCallback"
+    [resultTemplate]="nameTypeaheadTemplate"
+    [inputDisabled]="!!region"
+    (modelChange)="model.sites = getIdsFromAbstractModelArray($event)"
+  ></baw-typeahead-input>
+
+  <baw-typeahead-input
+    [label]="site ? (site?.isPoint ? 'Point(s)' : 'Site(s)') : 'Point(s)'"
+    [inputPlaceholder]="site ? site.name : 'All'"
+    [searchCallback]="siteSearchCallback"
+    [resultTemplate]="nameTypeaheadTemplate"
+    [inputDisabled]="!!site"
+    (modelChange)="model.points = getIdsFromAbstractModelArray($event)"
+  ></baw-typeahead-input>
+
+  <baw-typeahead-input
+    label="Recogniser(s)"
+    [inputPlaceholder]="'All'"
+    [searchCallback]="provenanceSearchCallback"
+    [resultTemplate]="nameTypeaheadTemplate"
+    (modelChange)="model.provenances = getIdsFromAbstractModelArray($event)"
+  ></baw-typeahead-input>
+
+  <label for="provenancesCutOffInput" class="form-label">
+    Recogniser Score Cut-Off
+  </label>
+  <input
+    #provenanceCutOffInput="ngModel"
+    [(ngModel)]="model.score"
+    id="provenancesCutOffInput"
+    name="provenancesCutOffInput"
+    type="number"
+    class="form-control"
+    placeholder="0"
+    step="0.01"
+    max="1"
+    min="0"
+  />
+
+  <label
+    *ngIf="provenanceCutOffInput.invalid && provenanceCutOffInput.dirty"
+    for="provenancesCutOffInput"
+    class="text-danger"
+  >
+    The recogniser score cut-off are outside the permitted boundary. Ensure that
+    the value is between 0 and 1.
+  </label>
```

Comment:
seems like a validation-messages component might be useful.  Future work?

```suggestion
  <validation-messages for="provenancesCutOffInput" />
```